examples/realtime/ftclient_rt_compute_psd.py

```diff
@@ -41,6 +42,10 @@
 
     n_fft = 256  # the FFT size. Ideally a power of 2
     n_samples = 2048  # time window on which to compute FFT
+
+    # make sure at least one epoch is available
+    time.sleep(float(n_samples) / raw_info['sfreq'])
```

Comment:
no need for `float` we should always have that `raw_info['sfreq']` is a float (if it's not, we have a bug elsewhere)